packages/ngircd/build.sh

```diff
@@ -0,0 +1,17 @@
+TERMUX_PKG_HOMEPAGE=https://ngircd.barton.de/
+TERMUX_PKG_DESCRIPTION="Free, portable and lightweight Internet Relay Chat server"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="@yonle <yonle@protonmail.com>"
+TERMUX_PKG_VERSION=26.1
+TERMUX_PKG_SRCURL="https://github.com/ngircd/ngircd/releases/download/rel-${TERMUX_PKG_VERSION}/ngircd-26.tar.xz"
+TERMUX_PKG_SHA256=56dcc6483058699fcdd8e54f5010eecee09824b93bad7ed5f18818e550d855c6
+TERMUX_PKG_DEPENDS="zlib"
```

Comment:
Archlinux has openssl as a dependency as well: https://archlinux.org/packages/community/x86_64/ngircd/

Reply:
Sure. I will add it